server/api.py

```diff
@@ -120,21 +123,40 @@ class Vcon(BaseModel):
 # Our local modules``
 sys.path.append("..")
 
+logging.config.fileConfig("./logging.conf")
 logger = init_logger(__name__)
 logger.info("Conserver starting up")
 
-
 # Load FastAPI app
 app = FastAPI.conserver_app
-
-app.mount("/static", StaticFiles(directory="static"), name="static")
 app.add_middleware(
     CORSMiddleware,
     allow_origins=["*"],
     allow_credentials=True,
     allow_methods=["*"],
     allow_headers=["*"],
 )
+app.add_middleware(
+    TrustedHostMiddleware,
+    allowed_hosts=["*"]
+)
+@app.middleware("http")
```

Comment:
Yes, I see that as well.   We can make that an option.   We implemented this because we were going to make the API public facing and we needed some way of securing it in the short term.

Reply:
We feel authentication should be enabled for now as a secure default. And add option to force disable the authentication method for certain situations. This can be added later as separate feature.